include/wil/filesystem.h

```diff
@@ -29,6 +30,7 @@ namespace wil
     //! Determines if a path is an extended length path that can be used to access paths longer than MAX_PATH.
     inline bool is_extended_length_path(_In_ PCWSTR path)
     {
+        assert(path != NULL);

```

Comment:
`_In_` presumes not-null, so I'm not entirely sure what this is trying to catch

Reply:
Yeah, I also already mentioned it in last sentence (Debug ver. or) via offering to add "if statement" to resolve it.